dotcom-rendering/src/lib/contributions.ts

```diff
@@ -4,13 +4,14 @@ import type { HeaderPayload } from '@guardian/support-dotcom-components/dist/dot
 import { useEffect, useState } from 'react';
 import type { DCRFrontType } from '../types/front';
 import type { DCRArticle } from '../types/frontend';
-// User Atributes API cookies (dropped on sign-in)
+
+// User Attributes API cookies (created on sign-in)
 export const HIDE_SUPPORT_MESSAGING_COOKIE = 'gu_hide_support_messaging';
 export const RECURRING_CONTRIBUTOR_COOKIE = 'gu_recurring_contributor';
 export const ONE_OFF_CONTRIBUTION_DATE_COOKIE = 'gu_one_off_contribution_date';
 export const OPT_OUT_OF_ARTICLE_COUNT_COOKIE = 'gu_article_count_opt_out';
 
-// Support Frontend cookies (dropped when contribution is made)
+// Support Frontend cookies (created when a contribution is made)
```

Comment:
good, I never know whether dropped means dropped into the browser or SQL type of DROP